packages/schemas/src/data-protection.ts

```diff
@@ -62,3 +62,13 @@ export const dataProtectionContentRequestedDataSchema = z.object({
 export const dataProtectionScanTriggeredWebhookDataSchema = z.object({
   organisationId: z.string().uuid(),
 });
+
+export  const dataProtectionDeleteObjectPermissionsRequestedDataSchema = z.object({
+  id: z.string().min(1),
+  organisationId: z.string().uuid(),
+  metadata: jsonSchema.optional(),
```

Comment:
```suggestion
  metadata: jsonSchema,
```

It's already optional.